app/models/metric/rollup.rb

```diff
@@ -3,9 +3,11 @@ module Metric::Rollup
                  [:stat_container_group_create_rate,
                   :stat_container_group_delete_rate,
                   :stat_container_image_registration_rate]
-  STORAGE_COLS = Metric.columns_hash.collect { |c, _h| c.to_sym if c.starts_with?("derived_storage_") }.compact
+  STORAGE_COLS = Metric.columns_hash.collect { |c, _h| c.to_sym if c.starts_with?("derived_storage_") }.compact.freeze
 
-  NON_STORAGE_ROLLUP_COLS = (ROLLUP_COLS - STORAGE_COLS)
+  NON_STORAGE_ROLLUP_COLS = (ROLLUP_COLS - STORAGE_COLS).freeze
+  BASIC_VM_CPU_COLS  = [:cpu_usage_rate_average, :derived_vm_numvcpus, :derived_memory_used, :net_usage_rate_average].freeze
+  BASIC_VM_DISK_COLS = [:cpu_usage_rate_average, :derived_memory_used, :disk_usage_rate_average, :net_usage_rate_average].freeze
```

Comment:
The first set is really around container usage, not vm usage. Also it includes more than CPU.  Also, also the pattern is to use the phrase ROLLUP_COLS (aside from storage), so I'm thinking rename to something like:

```suggestion
  CONTAINER_ROLLUP_COLS = [:cpu_usage_rate_average, :derived_vm_numvcpus, :derived_memory_used, :net_usage_rate_average].freeze
  VM_ROLLUP_COLS        = [:cpu_usage_rate_average, :derived_memory_used, :disk_usage_rate_average, :net_usage_rate_average].freeze
```